### PR TITLE
Redesign role points selector

### PR DESCRIPTION
**Pull request by Xaviju**
Check out redesign right here https://tree.taiga.io/project/taiga/issue/2108

### CODE REVIEW

**Comment by juanfran** [comment]
I can't see some letters fully (g, y..)

**Comment by Xaviju** [comment]
:-1:

**Comment by Xaviju** [comment]
:cool:

File: app/styles/layout/us-detail.scss [hunk 1]
```diff
@@ -289,15 +293,14 @@
         @extend %xlarge;
         @extend %title;
         display: block;
+        margin-bottom: .3rem;
         text-align: center;
     }
     .role {
```
Comment (Xaviju) [inline app/styles/layout/us-detail.scss]: test

**Pull request merged**